docs/README.md

```diff
@@ -0,0 +1,146 @@
+# Go API client for swagger
+
+Pureport API
+
+## Overview
+This API client was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  By using the [swagger-spec](https://github.com/swagger-api/swagger-spec) from a remote server, you can easily generate an API client.
+
+- API version: 1.0.0
+- Package version: 1.0.0
+- Build package: io.swagger.codegen.languages.GoClientCodegen
+
+## Installation
+Put the package under your project folder and add the following in import:
+```golang
+import "./swagger"
+```
+
+## Documentation for API Endpoints
+
+All URIs are relative to *http://api.pureport.com*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*AccountConsentApi* | [**Consent**](docs/AccountConsentApi.md#consent) | **Post** /accounts/{accountId}/consent | Consent to Master Service Agreement and Acceptable Use Policy for this account
+*AccountConsentApi* | [**HasConsent**](docs/AccountConsentApi.md#hasconsent) | **Get** /accounts/{accountId}/consent | Account consent to Master Service Agreement and Acceptable Use Policy
+*AccountInvitationsApi* | [**AcceptAllInvites**](docs/AccountInvitationsApi.md#acceptallinvites) | **Post** /invites | Accept all invites for the current user
+*AccountInvitationsApi* | [**Delete7**](docs/AccountInvitationsApi.md#delete7) | **Delete** /accounts/{accountId}/invites/{inviteId} | Delete account invite
```

Comment:
I thought you figured out a way to fix the "Delete7" (and other numbered) methods?

Reply:
Yeah, noticed that yesterday.
It looks like they are being set correctly for some resource end-points (*get*, *update*, ...) but not for *delete* for some reason:
```
    @ApiOperation("Delete account invite")
    @DELETE
    @RequiresAccountPermission(ResourceType.MEMBERS, Action.DELETE)
    open fun deleteAccountInvite() {
        accountInviteService.delete(accountInvite)
    }
```
Need to investigate that a bit further.

Reply:
https://pureport.atlassian.net/browse/PE-1042